testcafe/pages/ServicePage.js

```diff
@@ -1,16 +1,23 @@
 import { ReactSelector } from 'testcafe-react-selectors';
 import config from '../config';
 
-export default class ResourcePage {
+export default class ServicePage {
```

Comment:
This was a copy/paste error that the class was called `ResourcePage` when it was in the `ServicePage` file. It didn't error before because `export default` lets you import it with whatever name you want. Updated for consistency